python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -496,6 +496,8 @@ def update_index(self, this_id=None, verbose=False, firstRun=False):
                     ## we can't reasonably write all possible exceptions here!
                     logger.debug("update_index: Failed to load id %i: %s" % (this_id, str(x)))
                     summary.append((this_id, str(x)))
+                #finally:
+                #    pass
```

Comment:
Frankly I end up adding/removing these when testing the exceptions and there is far too much exception handling in opening the xml files to follow atm imo